create_framework/http_kernel_controller_resolver.rst

```diff
@@ -103,29 +103,29 @@ The ``index()`` method needs the Request object as an argument.
 ``getArguments()`` knows when to inject it properly if it is type-hinted
 correctly::
 
-    public function index(Request $request)
+    public function index(Request $request) { ... }
```

Comment:
Imho it is too verbose just to make the code checker happy. What about allowing it somehow as is?

Reply:
I agree. In the past (when this tool was introduced), we also decided that it's not the goal of documentation examples to be 100% syntax correct. Their goal is to explain the PHP API and do as good of a job as possible on not distracting the user with irrelevant details.

I'm -1 for these additions.